src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,47 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+
+  static removeDead() {
+    const aliveAnimals = this.alive.filter((animal) => animal.health > 0);
+
+    this.alive = [...aliveAnimals];
+  }
+
+  constructor(name, health) {
+    this.name = name;
+    this.health = health || 100;
+  }
 }
 
 class Herbivore extends Animal {
   // write your code here
+  constructor(health, name) {
+    super(health, name);
+    this.hidden = false;
```

Comment:
The constructor parameters for the Herbivore class are also in the wrong order. The name should be the first parameter, and health should be the second one, with a default value of 100 if not provided.